app/services/meursing_measure_finder_service.rb

```diff
@@ -28,6 +28,6 @@ def call
   attr_reader :root_measure, :additional_code_id
 
   def relevant_for_country?(measure)
-    measure.relevant_for_country?(root_measure.geographical_area_id)
+    measure.relevant_for_country?(root_measure.geographical_area)
```

Comment:
This shifts from using an attribute on `root_measure` to a record lookup - is this called in a loop anyway, ie will it introduce an N+1?

Reply:
Resolved this now :)